spec/Transaction.md

```diff
@@ -2,6 +2,8 @@
 
 Only a platform account can generate a transaction.
 The transaction generator pays the transaction fees.
+The transactions that cost less than the [minimum fee](Minimum-Fee.md) are rejected.
```

Comment:
Transactions that cost less than the [minimum fee](Minimum-Fee.md) are rejected.